src/libs/ReportUtils.ts

```diff
@@ -1247,54 +1332,52 @@ function getPersonalDetailsForAccountID(accountID) {
 
 /**
  * Get the displayName for a single report participant.
- *
- * @param {Number} accountID
- * @param {Boolean} [shouldUseShortForm]
- * @param {Boolean} shouldFallbackToHidden
- * @returns {String}
  */
-function getDisplayNameForParticipant(accountID, shouldUseShortForm = false, shouldFallbackToHidden = true) {
+function getDisplayNameForParticipant(accountID?: number, shouldUseShortForm = false, shouldFallbackToHidden = true): string | undefined {
     if (!accountID) {
         return '';
     }
     const personalDetails = getPersonalDetailsForAccountID(accountID);
     // this is to check if account is an invite/optimistically created one
     // and prevent from falling back to 'Hidden', so a correct value is shown
     // when searching for a new user
-    if (lodashGet(personalDetails, 'isOptimisticPersonalDetail') === true) {
+    if (personalDetails.isOptimisticPersonalDetail === true) {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
         return personalDetails.login || '';
     }
     const longName = personalDetails.displayName;
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
     const shortName = personalDetails.firstName || longName;
     if (!longName && !personalDetails.login && shouldFallbackToHidden) {
         return Localize.translateLocal('common.hidden');
     }
     return shouldUseShortForm ? shortName : longName;
 }
 
-/**
- * @param {Object} personalDetailsList
- * @param {Boolean} isMultipleParticipantReport
- * @param {Boolean} shouldFallbackToHidden
- * @returns {Array}
- */
-function getDisplayNamesWithTooltips(personalDetailsList, isMultipleParticipantReport, shouldFallbackToHidden = true) {
-    return _.chain(personalDetailsList)
-        .map((user) => {
+function getDisplayNamesWithTooltips(
+    personalDetailsList: PersonalDetails[] | Record<string, PersonalDetails>,
+    isMultipleParticipantReport: boolean,
+    shouldFallbackToHidden = true,
+): DisplayNameWithTooltips {
+    const personalDetailsListArray = Array.isArray(personalDetailsList) ? personalDetailsList : Object.values(personalDetailsList);
+
+    return personalDetailsListArray
+        ?.map((user) => {
```

Comment:
```suggestion
        .map((user) => {
```
I think optional chain is not required here